deployment/Jenkinsfile

```diff
@@ -0,0 +1,23 @@
+properties([pipelineTriggers([githubPush()])])
+
+pipeline {
+    agent any
+    stages {
+        stage('Checkout') {
+            steps {
+                checkout([$class: 'GitSCM', branches: [[name: '*/Dockerimages']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url:'https://github.com/tmarkesic/FakeRESTApiTests.git']]])
+                sh "ls -lart ./*"
```

Comment:
What is use case for this? If this was used during debugging process while developing, should be removed in final before merge.